spec/support/response_json.rb

```diff
@@ -0,0 +1,9 @@
+module ResponseJSON
```

Comment:
Even for acronyms, we use Tilecase not uppercase: https://github.com/IntrepidPursuits/web-style-guides/blob/master/style.md#naming